ios/Podfile.lock

```diff
@@ -206,6 +206,8 @@ PODS:
     - React
   - react-native-viewpager (3.3.0):
     - React
+  - react-native-webview (8.2.1):
```

Comment:
This should no longer be in your podfile. Please update your podfile and remove Podfile.lock / run `bundle exec pod install`.

Reply:
@efalkner This is out of scope of my PR here, Also its on the develop branch and used in LicensesScreen and News screen

Reply:
:) I just added a PR to remove it. It's automatically linked, and therefore doesn't need to be added to the Podfile.

It looks like your PR is putting the link back in, perhaps because it's out of date.

Reply:
@efalkner Its on the develop branch https://github.com/tripleblindmarket/covid-safe-paths/blob/develop/package.json 

Also as i said its been used in LicensesScreen and News screen, it will require refactor this part and its out of my PR scope

Reply:
Sorry for the confusion. I'm not asking you to remove the library from the project, just from your Podfile.lock. If you look at the diff, your PR is reintroducing that dependency to it.

Reply:
@efalkner Removed

Reply:
React-Native 60+ uses [auto linking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md). That means that for the majority of modules (react-native-webview included), it's not required to be linked manually. For iOS, that typically means adding an entry to the `Podfile`, which generates a `Podfile.lock` when you install cocoapods.

Given that this entry was not in the develop `Podfile.lock`, I can assume that the automatic linking was working properly in develop before your PR. That's why I'm thinking this got introduced unintentionally.

Reply:
^ we spoke offline, closing the loop :)